File: 250415/두 수의 합과 1/two-nums-sum-and-1.cpp
#include <iostream>
#include <string>
using namespace std;

int main()
{
    ios::sync_with_stdio(false);
    cin.tie(0);
    cout.tie(0);

    int n, m;
    cin >> n >> m;
    int sum = n + m;
    int count = 0;
    string result = to_string(sum);
    for(int i = 0; i < result.length(); i++) {
        if(result[i] == '1') {
            count++;
        }
    }
    cout << count;
    return 0;
}

File: 250415/두 수의 합과 1/two-nums-sum-and-1.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int a = sc.nextInt();
        int b = sc.nextInt();
        int sum = a + b;
        int count = 0;
        String s = Integer.toString(sum);
        for(int i = 0; i < s.length(); i++) {
            if(s.charAt(i) == '1'){
                count++;
            }
        }
        System.out.println(count);
        sc.close();
    }
}